### PR TITLE
introduce --quiet option

### LINKED ISSUE #1280: Support compose build --quiet
```
  -q, --quiet             Don't print anything to STDOUT

```
Also add the same flag to `config` and `pull` 
(note `docker-compose up` has `--quiet-pull` to disable output only related to pulling images)
We should also add this to the `up` command (but still attach to logs of requested)

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
introduce --quiet option

**Related issue**
close #1280

**(not mandatory) A picture of a cute animal, if possible in relation with what you did**
image

### CODE REVIEW

File: cli/cmd/compose/convert.go [hunk 1]
```diff
@@ -39,22 +43,33 @@ func convertCommand(p *projectOptions) *cobra.Command {
...
 		projectOptions: p,
 	}
-	convertCmd := &cobra.Command{
+	cmd := &cobra.Command{
 		Aliases: []string{"config"},
 		Use:     "convert SERVICES",
 		Short:   "Converts the compose file to platform's canonical format",
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if opts.quiet {
+				if opts.quiet {
```
Comment (chris-crone) [inline cli/cmd/compose/convert.go:52]: Guessing these shouldn't both be `opts.quiet`?

File: aci/compose.go [hunk 2]
```diff
@@ -100,7 +101,7 @@ func (cs aciComposeService) warnKeepVolumeOnDown(ctx context.Context, projectNam
 		if v.AzureFile == nil || v.AzureFile.StorageAccountName == nil || v.AzureFile.ShareName == nil {
 			continue
 		}
-		fmt.Printf("WARNING: fileshare \"%s/%s\" will NOT be deleted. Use 'docker volume rm' if you want to delete this volume\n",
+		fmt.Fprintf(os.Stderr, "WARNING: fileshare \"%s/%s\" will NOT be deleted. Use 'docker volume rm' if you want to delete this volume\n",
```
Comment (chris-crone) [inline aci/compose.go:104]: Do we still want to print out the stderr contents if we're in quiet mode? What does docker/cli do?

cc @thaJeztah

**Review by chris-crone — CHANGES_REQUESTED** [review]

File: cli/cmd/compose/convert.go (hunk 1 above)
Comment (ndeloof) [reply cli/cmd/compose/convert.go:52]: not sure what I had in mond dogin this ( ဖ ͜ စ)

File: aci/compose.go (hunk 2 above)
Comment (ndeloof) [reply aci/compose.go:104]: good point

File: cli/cmd/compose/convert.go (hunk 1 above)
Comment (chris-crone) [reply cli/cmd/compose/convert.go:52]: Making _really_ sure it's enabled :)

File: aci/compose.go (hunk 2 above)
Comment (thaJeztah) [reply aci/compose.go:104]: Hm... will have to check. Not sure what would be most expected. 🤔. I guess it's a "warning", so not a _critical_ message, but not sure how important it is to let the user know

**Review by aiordache — APPROVED** [review]

File: cli/cmd/compose/convert.go [hunk 3]
```diff
@@ -76,9 +89,14 @@ func runConvert(ctx context.Context, opts convertOptions, services []string) err
 	if err != nil {
 		return err
 	}
+	var out io.Writer = os.Stdout
 	if opts.Output != "" {
```
Comment (chris-crone) [inline cli/cmd/compose/convert.go:93]: Should this still work with quiet mode? The help says that it only does validation

**Review by chris-crone — APPROVED** [review]
LGTM with one question needing an answer

File: cli/cmd/compose/convert.go (hunk 3 above)
Comment (ndeloof) [reply cli/cmd/compose/convert.go:98]: Indeed. --ouput might be used with a path and then we would override file, and not just "validate and quit".
fixed

**Pull request merged**